repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -20,18 +21,31 @@
                              ])
 
 EVENT_TYPES = ('Present', 'Removed', 'Deprecated', 'Replaced', 'Split', 'Merged', 'Moved', 'Renamed')
-RELEASES = ((7, 5), (7, 6), (7, 7), (7, 8), (8, 0), (8, 1))  # TODO: bad, bad hardcode
+RELEASES = [(7, 5), (7, 6), (7, 7), (7, 8), (8, 0), (8, 1), (8, 2)]  # TODO: bad, bad hardcode
```

Comment:
can't you build that dynamically based on the information from the file?

Reply:
Hm, I could. It's one more list traversal but we won't have to update this later.

Reply:
Alright, it's a function now.